### PR TITLE
Add upgrade support

### PR DESCRIPTION
**Pull request by hrpatel**

### CODE REVIEW

**Comment by antevens** [comment]
+1

**Review by antevens — DISMISSED** [review]
+1

File: nginx-php-moodle/bin/configure-moodle.sh
Comment (mroote) [inline nginx-php-moodle/bin/configure-moodle.sh:86]: Do we need the force option here?

**Comment by hrpatel** [comment]
Sorry guys. Had to commit a change to get CI to pass - please review again.

**Comment by antevens** [comment]
I don't think you need another review/approval :D

File: nginx-php-moodle/bin/configure-moodle.sh
Comment (hrpatel) [reply nginx-php-moodle/bin/configure-moodle.sh:86]: Yeah, if the target file exists and its different a warning is logged and the file is overwritten

**Comment by hrpatel** [comment]
@antevens I configured GH to revoke an approval if commits are added after an approval

**Review by antevens — DISMISSED** [review]

**Review by mroote — APPROVED** [review]
+1

**Pull request merged**
